enterprise-suite/templates/prometheus.yaml

```diff
@@ -158,6 +158,19 @@ spec:
               scheme: HTTP
             initialDelaySeconds: 30
 
+        - name: es-monitor-api-configmap-reload
+          image: jimmidyson/configmap-reload:{{ .Values.configMapReloadVersion }}
+          resources:
+            requests:
+              cpu: {{ default .Values.defaultCPURequest .Values.configmapReloadCPURequest }}
+              memory: {{ default .Values.defaultMemoryRequest .Values.configmapReloadMemoryRequest }}
+          args:
+          - --volume-dir=/etc/config
```

Comment:
this is the shared config emptyDir for sharing config files between monitor api and prometheus.

The configMap is es-monitor-api-volume

Reply:
Shutting down on configMap updates seems ...ok... but there's this scenario where the reloader triggers every time after restart and we get in a death loop. How confident are you that can never happen? Just thinking out loud, a /reload endpoint would be softer as it wouldn't trigger pod recreation. I think all that endpoint would have to do is `	api.scheduleWriteConfig()`

Reply:
ConfigMaps don't sync more than every 60s in the first place. So I'm not worried about a death loop type deal. Reason I did it this way was to accommodate for changes to bootstrap monitors as well, without having to do a larger code change in es-monitor-api.

Reply:
Whoops "config-volume" name confused my brain.